src/cli.rs

```diff
@@ -52,5 +52,26 @@ pub(crate) enum Commands {
         target: Option<PathBuf>,
     },
     /// List components
+    Component {
+        #[command(subcommand)]
+        component: Component,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+pub(crate) enum Component {
     List {},
+    Create {
```

Comment:
Are all values args. At least the non optional should be options. This should be refined.